app/src/main/java/com/mychefassistant/presentation/ingredient/IngredientFragment.kt

```diff
@@ -34,11 +34,17 @@ class IngredientFragment : Fragment() {
         getArgs()
         title = view.findViewById(R.id.title)
 
-        ingredientViewModel.kitchen.observe(viewLifecycleOwner, Observer {
-            kitchen = it
-            loadedKitchen()
+        viewModel.event.observe(viewLifecycleOwner, Observer {
+            when (it) {
+                is Event.Info -> {
+                    when(it.type){
+                        IngredientViewModel.onKitchenLoad -> onKitchenLoad()

```

Comment:
What is this?

Reply:
you need [this](https://github.com/Kotlin/KEEP/pull/213)?